### PR TITLE
fix: do not memoize instance

### PR DESCRIPTION
**Pull request by raq929**
Causes table not to rerender with new data

### CODE REVIEW

File: src/DataTable/index.jsx [hunk 1]
```diff
@@ -73,19 +73,14 @@ function DataTable({
     // Stringifying the data gives a quick way of checking deep equality
   }, [fetchData, JSON.stringify(instance.state)]);
 
-  const enhancedInstance = useMemo(() => ({
+  const enhancedInstance = {
```
Comment (binodpant) [inline src/DataTable/index.jsx:76]: lol, the `instance` here is the react table instance right?

**Review by abutterworth — APPROVED** [review]
no need for useMemo?

**Comment by binodpant** [comment]
can you send me instructions to test locally? I think I sort of know module.config.js for this, but not 100%

File: src/DataTable/index.jsx (hunk 1 above)
Comment (raq929) [reply src/DataTable/index.jsx:76]: yep

**Comment by binodpant** [comment]
works like magic! it's fast too!!

**Pull request merged**